pskb_website/static/js/editor_utils.js

```diff
@@ -407,7 +407,7 @@ function openFullscreen() {
 function resizeEditor() {
     var lineHeight = editor.renderer.lineHeight;
     var maxLines = document.getElementById('editor-wrapper').offsetHeight / lineHeight;
-    editor.setOption('maxLines', Math.floor(maxLines) - 1);
```

Comment:
For the scroll sync to work with the past implementation, we need to set a high number in the maxLines. We had set something like 50000 or something. https://github.com/ajaxorg/ace/issues/2947